Source/Core/Runtime/Settings/SceneObjectTags.cs

```diff
@@ -43,16 +52,33 @@ public Guid Guid
                 }
             }
 
+            [SerializeField]
+            private TagType tagType;
+
+            /// <summary>
+            /// Type of tag showing weather it is a default "unique'ish" tag or a user defined tag.
+            /// </summary>
+            public TagType TagType => tagType;
```

Comment:
What is this used for? I remember talking about implementing it, but I see no references in code.

Reply:
The ides was to use it for identifying Unique and User Defined Tags E.g.: for filtering.

Reply:
I you are the opinion we will have no need for types anywhere else I will update the current method UpdateAvailableTags(List<SceneObjectTags.Tag> availableTags, SceneObjectTags.Tag uniqueTag = null) for adding a Unique Tags. Feels the more clean version.

Reply:
I found another place where it would be useful. In `SceneObjectTagDrawer.DrawSelectedTagsAndGameObjects` I distinguish Unique Tag by a `string.IsNullOrEmpty(label)`. By type would be better.